### DISCUSSION TITLE
JSBSim in MATLAB

### DISCUSSION THREAD

**Discussion by RaaYuGaa** [JSBSim-Team/jsbsim#848] [Support requests]
I am a first year student researching on aircraft collision and I need to build a simple aircraft model which does level flight at certain level with constant speed. I cannot figure out how to use JSBSim in MATLAB since I am new to coding.  I have downloaded JSBSim and Flightgear for now. Thank you.

**Comment by LeotaBacotk0** [comment]
Building an aircraft model can be challenging, but it's great that you're taking on this project as a first year student.

To use JSBSim in MATLAB, you'll first need to make sure that you have the necessary files and directories. Once you have downloaded JSBSim and Flightgear, you should be able to locate the JSBSim executable file, which you will need to run the simulation.

To call JSBSim from MATLAB, you can use the system command. For example, you might have something like this in your MATLAB script:

system('path_to_jsbsim\JSBSim.exe -script=path_to_script\my_script.xml');
In this example, you would replace path_to_jsbsim with the location of the JSBSim executable, and path_to_script with the location of your simulation script.

As for building your aircraft model, you'll need to define the physical characteristics of your aircraft, such as its mass, wing area, and aerodynamic properties. Once you have this information, you can use JSBSim to simulate the flight of your aircraft and analyze its behavior.

I hope this helps get you started! If you have any further questions, don't hesitate to ask.
